instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/servlet/AppServerBridge.java

```diff
@@ -19,34 +19,33 @@
       ContextKey.named("opentelemetry-servlet-app-server-bridge");
 
   public static Context init(Context ctx) {
-    return ctx.with(AppServerBridge.CONTEXT_KEY, new AppServerBridge());
+    return init(ctx, true);
+  }
+
+  public static Context init(Context ctx, boolean shouldRecordException) {
```

Comment:
I think javadoc here would help, to explain `shouldRecordException` (basically same good explanation as below on `shouldRecordException()` method)